amethyst_rendy/src/plugins.rs

```diff
@@ -0,0 +1,322 @@
+//! Set of predefined implementations of `RenderPlugin` for use with `RenderingBundle`.
```

Comment:
I'm torn about whether or not these should be in separate files. Would make it possible to see which bundles are available by just glancing at the files, and not having to scroll through this entire file.

Reply:
You probably mean what plugins. The way to see that in the docs is to navigate to `RenderPlugin` trait and see all implementers.

Reply:
Yeah, sorry about that, meant plugins. I'm still not sure about this, I don't really see any benefit to having them in the same file, if there are more or if they expand over time, we would have to break them up anyways...